village/src/components/Smurfs.js

```diff
@@ -1,9 +1,26 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
+  state = {
+    smurfs: [],
+    name: "",
```

Comment:
We don't need to keep `name`, `age` or `height` in our state here. Those keep track of our values in the `<input>` form in `SmurfForm.js`.